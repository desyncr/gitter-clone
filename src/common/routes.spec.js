import React       from 'react';
import { shallow } from 'enzyme';
import toJson      from 'enzyme-to-json';
import Routes,
  { routes }       from './routes';

describe('routes', () => {
  it('should match snapshot', () => {
    expect(routes).toMatchSnapshot();
  });
});

describe('<Routes />', () => {
  it('should render correctly', () => {
    const component = <Routes />;
    const wrapper = shallow(component);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
