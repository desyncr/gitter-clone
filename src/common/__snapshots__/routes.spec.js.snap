// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> should render correctly 1`] = `
<Router
  history={
    Object {
      "createHref": [Function],
      "createKey": [Function],
      "createLocation": [Function],
      "createPath": [Function],
      "getCurrentLocation": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "listenBefore": [Function],
      "push": [Function],
      "replace": [Function],
      "transitionTo": [Function],
    }
  }
  render={[Function]}
>
  <Route
    component={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
    path="/"
  >
    <IndexRoute
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
    />
    <Route
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
      path="/explore"
    />
    <Route
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
      path="/:convo"
    />
  </Route>
  <Route
    component={[Function]}
    path="*"
  />
</Router>
`;

exports[`routes should match snapshot 1`] = `
<Router
  history={
    Object {
      "createHref": [Function],
      "createKey": [Function],
      "createLocation": [Function],
      "createPath": [Function],
      "getCurrentLocation": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "listenBefore": [Function],
      "push": [Function],
      "replace": [Function],
      "transitionTo": [Function],
    }
  }
  render={[Function]}
>
  <Route
    component={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
    path="/"
  >
    <IndexRoute
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
    />
    <Route
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
      path="/explore"
    />
    <Route
      components={
        Object {
          "main": [Function],
          "menu": [Function],
        }
      }
      path="/:convo"
    />
  </Route>
  <Route
    component={[Function]}
    path="*"
  />
</Router>
`;
